UndertaleModTool/CommunityScripts/FancyRoomSelect.csx

```diff
@@ -411,7 +408,7 @@ exiting = 0
 ", Data);
 
 
-ScriptMessage("Successfully applied for Gamemaker " + (gms2 ? 2 : 1).ToString() + @"
+ScriptMessage("Successfully applied for Gamemaker " + (Data.GMS2_3 ? "2.3" : (gms2 ? "2" : "1")) + @"
```

Comment:
Would personally prefer if the GM variable would get split into seperate statement(s), rather than trying to force it into the scriptMessage here.